locales/en.json

```diff
@@ -377,6 +377,31 @@
     "youSuccessfullyClaimedNft": "You successfully minted {0} NFT",
     "yourWalletNeeds": "Your wallet needs a minimum of {0} to claim this drop."
   },
+  "edit": {
+    "collection": {
+      "banner": {
+        "hint": "1440 x 560 size recommended",
+        "label": "Banner",
+        "message": "Represents your collection on its page and on collection card"
+      },
+      "drop": "Don't click, just drop!",
```

Comment:
any reason for this one?

Reply:
from design 

![CleanShot 2024-11-20 at 14 29 57@2x](https://github.com/user-attachments/assets/5999f5ca-ea49-4378-8525-fdef2be70831)